docs/docs/module_guides/observability/index.md

```diff
@@ -356,6 +356,32 @@ openlit.init()
 
 - [OpenLIT's Official Documentation](https://docs.openlit.io/latest/integrations/llama-index)
 
+### AgentOps
+
+[AgentOps](https://github.com/AgentOps-AI/agentops) helps developers build, evaluate,
+and monitor AI agents. AgentOps will help build agents from prototype to production,
+enabling agent monitoring, LLM cost tracking, benchmarking, and more.
+
+#### Install
+
+```shell
+pip install agentops
```

Comment:
I guess it would be `pip install llama-index-instrumentation-agentops` (?)

Reply:
Yes I think so! Fixed